website/docs/guides/dbt-ecosystem/sl-partner-integration-guide.md

```diff
@@ -0,0 +1,671 @@
+---
+title: "dbt Semantic Layer integration" 
+id: "sl-partner-integration-guide"
+description: Learn about partner integration guidelines, roadmap, and connectivity. 
+---
+
+# dbt Semantic Layer partner integration
+
+:::info Contact us
+
+This guide is for dbt Semantic Layer integration partners and explains integration guidelines, product roadmap, and connectivity.<br />
+
+To become a formal partner, integrate with the API, or have questions/feedback &mdash; **[contact us](mailto:semantic-layer@dbtlabs.com)** for more info.
+
+:::
+
+
+## Overview
+
+The dbt Semantic Layer allows users to dynamically generate and query datasets in downstream tools based on their dbt governed assets, such as metrics, models, and entities. It helps organizations manage complexities such as data, tools, and teams to make more efficient and trustworthy decisions.
+
+The rapid growth of different tools in the modern data stack has helped data professionals. The various tools help solve the diverse needs of different teams. The downside of this growth is the fragmentation of business logic across teams, tools, and workloads. 
+
+To solve this, the dbt Semantic Layer provides a platform where users can confidently leverage their data from within their tools. dbt Cloud's change management capabilities ensure that any user modifications made to core business constructs, like metrics or entities, are distributed into all the tools connected to the data platform.
+
+The dbt Semantic Layer is unopinionated about specific tools, use cases, or applications of data. The following summarizes some of the common use cases:
+
+* Business intelligence (BI), reporting, and analytics,
+* Data quality and monitoring,
+* Governance and privacy,
+* Data discovery and cataloging,
+* Machine learning and data science.
+
+<!-- rewrite below little and would like to see if i can include a button or callout -->
+
+:::info Share your use case
+
+If you'd like to share other use cases for the dbt Semantic Layer, contact the [dbt Labs team](mailto:semantic-layer@dbtlabs.com).
+
+:::
+
+
+## Product overview
+
+The dbt Semantic Layer product architecture includes four primary components:
+
+| Components | Information | Developer plans | Team plans | Enterprise plans | License |
+| --- | --- | :---: | :---: | :---: | --- |
+| **[dbt Project](/docs/build/metrics)** | Define models and metrics in dbt Core. | ✅ | ✅ |  ✅  | Open source in dbt Core |
```

Comment:
Should this be:
```suggestion
| **[dbt Metrics](/docs/build/metrics)** | Define models and metrics in dbt Core. | ✅ | ✅ |  ✅  | Open source in dbt Core |
```

Reply:
ah i think @azzam34 @callum-mcdata or @cafzal said this should be project. i can't remember who so tagging them to confirm for me.